client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -20,6 +20,13 @@ import { getUnconnectedSiteUrl } from 'state/selectors';
 import { JETPACK_ONBOARDING_STEPS as STEPS } from '../constants';
 
 class JetpackOnboardingSummaryStep extends React.PureComponent {
+	handleNextStepSelection = stepName => () => {
```

Comment:
Calling this will produce for example `site-title` for the completed steps and `BLOG` for next steps - doesn't seem very consistent, should we alter it?

Should be fine if we have separate events for completed and next steps (see my other comments).